### ISSUE TITLE
Default rrdtool create & script "tune_port.php" should be changed.

### ISSUE THREAD

**Issue by NightowlKr** [librenms/librenms#14582]
The point of this issue is that the graph data is strange.
----
![1](https://user-images.githubusercontent.com/5241111/200065574-5c71539d-6f6f-4a4b-a3a9-20912f6872ff.png)

Because it's the sum graph for the two ports below.
----
![2](https://user-images.githubusercontent.com/5241111/200065586-e4acf1c4-fdd8-4e95-8cbd-c462c4a83543.png)
![3](https://user-images.githubusercontent.com/5241111/200065679-0dbe60e3-92db-46d5-a349-53acb148961e.png)

Moreover, the total graph was also strange.
----
![4](https://user-images.githubusercontent.com/5241111/200065609-696ec09b-b6c9-4c00-babc-8f6d8b53751c.png)

I checked that the collection was working fine.
----
![5](https://user-images.githubusercontent.com/5241111/200065843-27812f3b-2844-4b02-a254-a680d15e954a.png)

So, i found the point of failure.
----
![6](https://user-images.githubusercontent.com/5241111/200065853-79a2389e-1aae-4708-bfe8-cd664a3c5634.png)

Eventually, i fount no data in the point at rrd file.
----
![7](https://user-images.githubusercontent.com/5241111/200065878-6e16690c-b811-4b1f-a13c-8816ababe91f.png)

When i looked at the structure of the rrd file,  it seemed strange.
Because according to the document below, it had to be supported up to 100Gbps.
     https://docs.librenms.org/Extensions/RRDTune/
----

![8](https://user-images.githubusercontent.com/5241111/200065950-8cd75b42-b2a6-4021-ad94-a7dc27ac4dc5.png)

It was the same even if the corresponding php was executed manually.
----
![10](https://user-images.githubusercontent.com/5241111/200065966-31890230-c6a7-4442-8564-c94e1dadfb17.png)

![11](https://user-images.githubusercontent.com/5241111/200065981-b4d76b60-7547-4e21-91c1-4a014d487231.png)

Finally, to resolve this symptom, I changed it manually by referring to the URL below.
https://oss.oetiker.ch/rrdtool/doc/rrdtune.en.html#:~:text=disable%20this%20limit.-,%2D%2Dmaximum%7C%2Da%C2%A0ds%2Dname%3Amax,-alter%20the%20maximum
----
![12](https://user-images.githubusercontent.com/5241111/200066004-d324c29b-9623-44cf-af05-c97b94597d77.png)

### Output of ./validate.php

```txt
===========================================
Component | Version
--------- | -------
LibreNMS  | 22.10.0-104-g89698ed59 (2022-11-05T00:03:46+09:00)
DB Schema | 2022_08_15_084507_add_rrd_type_to_wireless_sensors_table (248)
PHP       | 8.1.12
Python    | 3.8.10
Database  | MariaDB 10.6.10-MariaDB-1:10.6.10+maria~ubu2004
RRDTool   | 1.7.2
SNMP      | 5.8
===========================================

[OK]    Composer Version: 2.4.4
[OK]    Dependencies up-to-date.
[OK]    Database connection successful
[OK]    Database Schema is current
[OK]    SQL Server meets minimum requirements
[OK]    lower_case_table_names is enabled
[OK]    MySQL engine is optimal
[OK]
[OK]    Database schema correct
[OK]    MySQl and PHP time match
[OK]    Active pollers found
[OK]    Dispatcher Service not detected
[OK]    Locks are functional
[OK]    Python poller wrapper is polling
[OK]    Redis is unavailable
[OK]    rrd_dir is writable
[OK]    rrdtool version ok
[FAIL]  We have found some files that are owned by a different user than 'librenms', this will stop you updating automatically and / or rrd files being updated causing graphs to fail.
        [FIX]:
        sudo chown -R librenms:librenms /opt/librenms
        sudo setfacl -d -m g::rwx /opt/librenms/rrd /opt/librenms/logs /opt/librenms/bootstrap/cache/ /opt/librenms/storage/
        sudo chmod -R ug=rwX /opt/librenms/rrd /opt/librenms/logs /op
        Files:
         /opt/librenms/rrd/smokeping/Ungrouped/~~~~~~~~~.rrd
         /opt/librenms/rrd/smokeping/network/~~~~~~~~.rrd
         /opt/librenms/rrd/smokeping/__sortercache/data.lnmsFPing-1.storable
         /opt/librenms/rrd/smokeping/__sortercache/data.lnmsFPing-0.storable
```

### What was the last working version of LibreNMS?

22.10.0

**Comment by murrant** [comment]
You hit the max default value for port bits.  The reason it has a max is because bad values either from the device (or network interruptions) can cause spikes.

@librenms/reviewers Thoughts on the max value? 10G is very common these days and 40G and 100G+ are becoming more common.

1. Remove the max, let the spikes be very large.
2. Increase the max, we still care about spikes, but want to minimize user adjustments.

**Comment by NightowlKr** [comment]
@murrant 
According to your opinion, I set it to 800G (800 * 1000 * 1000 * 1000 / 8 = 100000000000).
Because recently standards  is QSFP-DD800.
    http://www.qsfp-dd.com/specification/

**Comment by murrant** [comment]
If we go with option 2, that would be fine for most people for awhile.

**Comment by ottorei** [comment]
> If we go with option 2, that would be fine for most people for awhile.

Agree, the value should probably be at least a few hundred gigs since 100G ports are becoming quite popular on newer hardware.

**Comment by paulgear** [comment]
I guess the issue is which is more common: buggy SNMP implementations causing spikes, or hitting the default max?  Going forward it will probably be more and more likely to hit the max, so bumping it seems like a good choice.  However, a 5 Gbps spike on my VDSL uplink at home is still going to show up as a huge anomaly, so people will likely have to manually set the value lower on some ports anyway.  I'd probably vote for bumping it to 100 Gbps by default so that people don't end up with bad data in their RRDs.

**Comment by murrant** [comment]
@paulgear I think LibreNMS could do a better job of detecting snmp queries interrupted mid-query and prevent writing 0s to the rrd.  That would help the spikes a lot.

Unfortunately, that involves refactoring the ports module which is a huge monster containing all kinds of black magic.
